apps/studio/components/ui/AIAssistantPanel/AIAssistant.tsx

```diff
@@ -168,6 +181,7 @@ export const AIAssistant = ({ className }: AIAssistantProps) => {
             key={message.id}
             message={message}
             isLoading={isChatLoading && message.id === chatMessages[chatMessages.length - 1].id}
+            onResults={(props) => snap.updateMessage({ id: message.id, ...props })}
```

Comment:
Since you're using a memoized component here, this should also be memo'd so it doesn't break the memoization – wrap it in `useCallback` :)